lucien-ios/Extensions/UIViewController+Extensions.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  UIViewController.swift
+//  lucien-ios
+//
+//  Created by Fang, Gracie on 10/4/17.
+//  Copyright © 2017 Intrepid Pursuits. All rights reserved.
+//
+
+import Foundation
+
+extension UIViewController {
+
+}
+
+extension UILabel {
```

Comment:
This shouldn't be in this file, it's an extension on `UILabel` so it should be in a file named `UILabel+Extensions`